src/kixi/gwyn/gwyn.clj

```diff
@@ -39,19 +41,56 @@
    (-> fire-station-lookup-table
        (wds/select-from-ds {:station fire-station})
        (wds/subset-ds :cols [:radius :lat :long]))})
+
+(def google-places-api "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=")
+
+(def api-key "&key=AIzaSyB4eWT7aEhsik-rP813fNJIN-qo12nhgIM")
+
+(defn fetch-url [url]
+  (html/html-resource (java.net.URL. url)))
+
+(defn api-str [loc]
+  (str google-places-api loc api-key))
+
+(defn parse-google-places [api-call]
+  (-> api-call
+      fetch-url
+      first
+      :content
+      first
+      :content
+      first
+      json/read-str
+      (get "results")))
```

Comment:
There's quite a lot of potential for this to go pop if the result from Google is not what you expect. You should check that it's first returned you an 'OK' (status = 200) result before starting to dive into it.